docs/how_to/apphooks.rst

```diff
@@ -8,7 +8,7 @@ this case, you can create a normal django CMS page without any content of its
 own, and attach the news application to the page; the news application's content
 will be delivered at the page's URL.
 
-To create an apphook place a ``cms_apps.py`` in your application. And in it write
+To create an Apphook place a ``cms_app.pys`` in your application. And in it write
```

Comment:
@evildmp this renaming changes ``cms_app.py`` to ``cms_apps.py``, the documentation has been changed accordingly